lightcycle-integration-test/src/main/java/com/soundcloud/lightcycle/sample/basic/AppCompatActivityLogger.java

```diff
@@ -0,0 +1,63 @@
+package com.soundcloud.lightcycle.sample.basic;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+
+import com.soundcloud.lightcycle.sample.basic.callback.ActivityCallback;
+import com.soundcloud.lightcycle.sample.basic.callback.ActivityLifecycleCallback;
+
+class AppCompatActivityLogger extends BaseActivityLogger<AppCompatActivity> {
```

Comment:
I thought we can use DefaultActivityLightCycle<AppCompatActivity> like what you use in [here](https://github.com/soundcloud/lightcycle/blob/master/examples/basic/src/main/java/com/soundcloud/lightcycle/sample/basic/ActivityLogger.java#L10)?